test/e2e/suites/unmanaged/unmanaged_CAPI_clusterclass_test.go

```diff
@@ -41,7 +41,7 @@ var _ = ginkgo.Context("[unmanaged] [Cluster API Framework] [ClusterClass]", fun
 		Expect(e2eCtx.Environment.BootstrapClusterProxy).ToNot(BeNil(), "Invalid argument. BootstrapClusterProxy can't be nil")
 	})
 
-	ginkgo.Describe("Self Hosted Spec [ClusterClass]", func() {
+	ginkgo.PDescribe("Self Hosted Spec [ClusterClass]", func() {
```

Comment:
Are those still not working or can they be changed back to `Describe` (not pending)? If they fail, can we add a comment?

[repeated]

Reply:
See https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/5169

Reply:
I also created #5176 to review all other tests marked pending.